### ISSUE TITLE
Install failed on Windows

### ISSUE THREAD

**Issue by Meeeeow** [pnpm/pnpm#741]
### Code to reproduce the issue:
`pnpm i`

### Expected behavior:
```
ERROR EXDEV: cross-device link not permitted, link 'C:\Users\User\.pnpm-store\1\.registry.npmjs.org+16384\babel-core\6.24.1\README.md' -> 'D:\proj\node_modules\.registry.npmjs.org+16384\babel-core\6.24.1\node_modules\babel-core+stage\README.md'
```

 - `node -v` prints: 7.9.0
 - Windows, OS X, or Linux?: Windows 10 Version 1703

**Comment by vjpr** [comment]
See https://github.com/pnpm/pnpm/issues/712

**Comment by zkochan** [comment]
Fix published in [v1.15.0](https://github.com/pnpm/pnpm/releases/tag/v1.15.0). Each disk has a dedicated store now.
